packages/account/src/Components/poi/idv-status/idv-failed/idv-failed.tsx

```diff
@@ -0,0 +1,395 @@
+        return <Loading is_fullscreen={false} className='account__initial-loader' />;
+    }
+
+    return (
+        <Formik
+            initialValues={rest_state?.form_initial_values ?? {}}
+            enableReinitialize
+            onSubmit={onSubmit}
+            validate={validateFields}
+            className='proof-of-identity__container'
+        >
+            {({ isSubmitting, isValid, dirty }) => (
+                <Form
+                    className={classNames('proof-of-identity__mismatch-container', {
+                        'upload-layout': is_document_upload_required,
+                    })}
+                >
+                    <FormBody className='form-body'>
+                        <Text
+                            size={isMobile() ? 'xs' : 's'}
+                            weight='bold'
+                            className='proof-of-identity__failed-warning'
+                            align='center'
+                        >
+                            <Localize i18n_default_text='Your identity verification failed because:' />
+                        </Text>
+                        <HintBox
+                            icon='IcCloseCircleRed'
+                            icon_height={16}
+                            icon_width={16}
+                            message={
+                                <Text as='p' size={isMobile() ? 'xxs' : 'xs'} data-testid={mismatch_status}>
+                                    {idv_failure?.failure_message}
+                                </Text>
+                            }
+                            is_danger
+                        />
+                        {is_document_upload_required && (
+                            <React.Fragment>
+                                <Text
+                                    size='xs'
+                                    className='proof-of-identity__failed-warning'
+                                    align={isMobile() ? 'left' : 'center'}
+                                >
+                                    <Localize i18n_default_text='Let’s try again. Choose another document and enter the corresponding details.' />
+                                </Text>
+                                <FormSubHeader title={localize('Identity verification')} />
+                                <IDVForm
+                                    selected_country={selected_country}
+                                    hide_hint={true}
+                                    class_name='idv-layout idv-resubmit'
+                                />
+                                <FormSubHeader title={localize('Details')} />
+                            </React.Fragment>
+                        )}
+                        <PersonalDetailsForm
+                            editable_fields={rest_state?.changeable_fields}
+                            is_qualified_for_idv
```

Comment:
where is this value coming from?

Reply:
The flag `is_qualified_for_idv` is a flag we are using in the shared Personal-details-form. In other places it is based on verifying if the country is IDV supported. But in case of IDV-failed component, since it renders only for IDV supported countries, we are setting it to true